rpc/amqp-rpc.go

```diff
@@ -171,7 +173,7 @@ type AmqpRPCServer struct {
 
 // NewAmqpRPCServer creates a new RPC server for the given queue and will begin
 // consuming requests from the queue. To start the server you must call Start().
-func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer)) (*AmqpRPCServer, error) {
+func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer), durable, autoAck bool) (*AmqpRPCServer, error) {
```

Comment:
@jcjones proposed that the Publisher RPC queue be durable to make life somewhat easier in terms of needing a tool that checks for certificates with SCT submissions that have failed. I'm happy to split this out for now, perhaps specifically until the missing SCT scanner exists and we actually care about using the SCTs for OCSP...